zip-0320.rst

```diff
@@ -200,19 +200,8 @@ text:
 
 will be replaced by:
 
-  A Unified Address or Unified Viewing Key MUST **either**:
-
-  * contain at least one shielded Item (Typecodes :math:`\mathtt{0x02}`
-    and :math:`\mathtt{0x03}`), and **no** Source Restriction Metadata Item
-    having value :math:`\mathtt{0x00}`; **or**
-  * contain **both** a Transparent Item (Typecode :math:`\mathtt{0x00}` or
-    Typecode :math:`\mathtt{0x01}`) **and** a Source Restriction Metadata Item
-    (Typecode :math:`\mathtt{0xE2}`) having value :math:`\mathtt{0x00}`, and
-    **no other** non-Metadata Items.
-
-In the following, the “``u+``” Human Readable Part of Revision 1 Unified
-Addresses is a placeholder value, pending the finalization of Revision 1 
-[#zip-0316-revision-1]_.
+  A Unified Address MUST contain at least one Receiver and any number
+  of Metadata Items.
```

Comment:
This should be expressed as ZIP 320 depending on the fact that r1 UAs allow a single transparent receiver, rather than as an edit.